### PR TITLE
test-Bonheure-Antoine

### PR DESCRIPTION
**Pull request by BonheureAntoine**

### CODE REVIEW

**Comment by iglesias1602** [comment]
I, the greatest cat will review this pull request

**Review by iglesias1602 — CHANGES_REQUESTED** [review]
- Both tests work fine just a few things to say: 
- There is no explanation of the modifications that were made to the file in the commit
- There is one mock test that was created inside the class TestSimpleBin(unittest.TestCase) but outside there one extra test, it seems it is a miss click, because the test inside the test_get_snippet() it's the same

File: simplebin.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def status():
 def new():
     return bottle.template('newbin')  # newbin.tpl
 
-@bottle.route('/<name>')
+@bottle.route('/<name>')
```
Comment (Julien00859) [inline simplebin.py:59]: useless diff

File: tests.py [hunk 2]
```diff
@@ -16,6 +17,13 @@ def urlopen(path):
 
 
 class TestSimpleBin(unittest.TestCase):
+
+
+    with unittest.mock.patch('simplebin.Snippet') as mock_snippet:
+         mock_snippet.get_by_name.side_effect = ValueError("Cannot create a new unique id.")
+
+
+
```
Comment (Julien00859) [inline tests.py:26]: as noted by @iglesias1602 this seems to be deadcode

**Review by Julien00859 — CHANGES_REQUESTED** [review]
The commit title/message isn't up to our expectation, imagine reading the git history in a few years and wondering what this "tests" commit do?

Please change your commit title so it can finish the following sentence: "Thanks to this commit we can...". Also please provide a commit body where you explain *why* those changes are necessary, you can do a short summary of the issue. Also please reference the issues you are fixing, if any.

Don't forget to rebase your branch on the latest master too

**Pull request closed without merging**